### PR TITLE
rework of some aspects of the paper

### PR DESCRIPTION
**Pull request by dogoepp**
These modifications affect the paper more deeply than my former proposals. It might be desirable to cherry pick commits on their own, as some modifications might not be accepted.

I made sure that each commit carries a well separated modification.

### CODE REVIEW

File: workflow/scripts/image_cache_bin.R [hunk 1]
```diff
@@ -14,7 +14,7 @@ df <- read_csv(filename, col_names = T) %>%
     "Image in binary cache?" = bin_cache,
     "Long-term binary cache?" = bin_cache_long_term,
     "Image recipe available?" = recipe_in_repo,
-    "Long-term binary cache or recipe" = long_term_safe
+    "Long-term binary cache / recipe?" = long_term_safe
```
Comment (GuilloteauQ) [inline workflow/scripts/image_cache_bin.R:17]: keep `?` and `or`

File: rep24/main.tex [hunk 2]
```diff
@@ -779,24 +781,27 @@ \section{Towards a new badge for artifacts}\label{sec:longevity}
...
 
-Artifacts with long \emph{longevity} are much more valuable and impactful for future researchers to extend, and deserve to be rewarded and have more visibility.
+Artifacts with long \emph{longevity} are much more valuable and impactful as they allow for future researchers to validate the results and build on former research.
+They deserve to be rewarded and have more visibility.
 Table\ \ref{tab:longevity} proposes a grading framework to evaluate the \emph{longevity} of an artifact.
-The artifact is evaluated on three criteria: sharing of the source code, experimental setup used, and software environment. 
-We propose of each criteria 4 levels going from poor but passable quality (1/4) to better (5/4).
+The artifact is evaluated on three \emph{longevity} criteria: how the source code is shared, the availability of the experimental setup, and the longevity of the software environment itself.
+We propose to grade each aspect on 5 levels going from missing (0/4) to best (4/4).
```
Comment (GuilloteauQ) [inline rep24/main.tex:790]: not sure about "missing"

File: rep24/main.tex [hunk 3]
```diff
@@ -779,24 +781,27 @@ \section{Towards a new badge for artifacts}\label{sec:longevity}
...
+As we have seen in Section\ \ref{sec:sop}, the popular tools and methods to share source code, package software environment, or platforms to perform experiments on differ in their \emph{longevity} guarantees/quality.
 
-Artifacts with long \emph{longevity} are much more valuable and impactful for future researchers to extend, and deserve to be rewarded and have more visibility.
+Artifacts with long \emph{longevity} are much more valuable and impactful as they allow for future researchers to validate the results and build on former research.
+They deserve to be rewarded and have more visibility.
 Table\ \ref{tab:longevity} proposes a grading framework to evaluate the \emph{longevity} of an artifact.
-The artifact is evaluated on three criteria: sharing of the source code, experimental setup used, and software environment. 
-We propose of each criteria 4 levels going from poor but passable quality (1/4) to better (5/4).
+The artifact is evaluated on three \emph{longevity} criteria: how the source code is shared, the availability of the experimental setup, and the longevity of the software environment itself.
```
Comment (GuilloteauQ) [inline rep24/main.tex:789]: \emph{longevity}

File: rep24/main.tex [hunk 4]
```diff
@@ -125,11 +125,11 @@
...
+  However, the notion of "reproducibility" considered by the badges is limited, and it misses important aspects of the reproducibility problem.
   In this paper, we surveyed 296 papers from 5 top conferences in system and distributed systems of 2023.
   For each paper of these conferences, we gathered information about its artifacts (how it was shared, which experimental setup, and how the software environment was generated and shared), as well as the badges awarded.
-  We concluded that the state-of-the-practice does not address the problems of reproducibility in terms of \emph{longevity} of the artifacts.
-  We thus propose a new badge to reward artifacts that will resist the test of time, and associate with this new badge, a framework for grading and delivering or not the badge to authors.
+  We conclude that the state-of-the-practice does not address the problems of reproducibility in terms of \emph{longevity} of the artifacts.
+  To start a discussion in the community, we propose a new badge to reward artifacts that will resist the test of time, based on three criteria.
```
Comment (GuilloteauQ) [inline rep24/main.tex:132]: "with the community" ? "Communities" ?

File: rep24/main.tex (hunk 2 above)
Comment (dogoepp) [reply rep24/main.tex:790]: neither am I. I did not dare write "unsatisfactory" or "inadequate" or "insufficient" as it seemed too harsh.

Comment (GuilloteauQ) [reply rep24/main.tex:790]: i vote "insufficient"

**Comment by GuilloteauQ** [comment]
awesome!

**Pull request merged**